app/models/mets_document.rb

```diff
@@ -36,6 +36,12 @@ def right_to_left
     @mets.xpath("/mets:mets/mets:structMap[@TYPE='Physical']/mets:div/@TYPE").to_s.start_with? 'RTL'
   end
 
+  def viewing_hint
+    type = @mets.xpath("/mets:mets/mets:structMap[@TYPE='Physical']/mets:div/@TYPE").to_s
+    return if ['TightBoundManuscript', 'ScrollSet', 'BoundArt'].any? { |w| type.include?(w) }
```

Comment:
Because we're doing `type.include?('TightBoundManuscript')` anything with that substring will pass, so `RTLTightBoundManuscript`, `FooTightBoundManuscript`, `TightBoundManuscriptsAreCool` etc.